source/builtins.rst

```diff
@@ -215,8 +215,10 @@ The call::
 
 was equivalent to::
 
+    import tokenize
+
     def compile_file(filename):
-        with open(filename) as f:
+        with open(filename, encoding='utf-8') as f:
```

Comment:
If you want to use `encoding` kwarg in `open()` function in Python 2, you have to import `open()` function from `io` module first. Default `open()` function in Python 2 doesn't support that.